lms/product/factory.py

```diff
@@ -14,7 +15,7 @@ def get_product_from_request(request) -> Product:
     return product
 
 
-def _get_family(request):
+def _get_family(request):  # pylint:disable=too-many-return-statements
```

Comment:
This could be refactor to avoid pylint's complain but I think the method reads well as it is, try different methods to get the family, early return when one works.